en/docs/get-started/how-to-guides/ai-code-generation.md

```diff
@@ -0,0 +1,165 @@
+# AI Code Generation
```

Comment:
Suggestion : Isn't it better to change this as AI Integration Generation since focus is on genrating integrations with AI?